controlplane/hack/deploy/addon/policy/hub/crds/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+resources:
+- apps.open-cluster-management.io_placementrules_crd.yaml
```

Comment:
need to ignore the placementrule. Right now, we will use placement instead. we won't install placementrule controller upon the standalone controlplane.

Reply:
the policy propagator will be on crash loop if the placementrule CRD is missing.

Reply:
I see. Let me have this CRD temporarily.